src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -1,14 +1,23 @@
 /** @jsx jsx */
-import { ipc_clearDolphinCache, ipc_configureDolphin, ipc_reinstallDolphin } from "@dolphin/ipc";
+import {
+  ipc_clearDolphinCache,
+  ipc_configureDolphin,
+  ipc_importDolphinSettings,
+  ipc_reinstallDolphin,
+} from "@dolphin/ipc";
 import { DolphinLaunchType } from "@dolphin/types";
 import { css, jsx } from "@emotion/react";
 import Button from "@material-ui/core/Button";
 import CircularProgress from "@material-ui/core/CircularProgress";
+import FormHelperText from "@material-ui/core/FormHelperText";
 import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
 import Typography from "@material-ui/core/Typography";
-import { isLinux } from "common/constants";
+import { isLinux, isMac } from "common/constants";
 import { shell } from "electron";
+import log from "electron-log";
+import _ from "lodash";
```

Comment:
```suggestion
import capitalize from "lodash/capitalize";
```

Can we just use the default capitalize import if it's the only function we're using from lodash?